instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/instrumentation.rb

```diff
@@ -46,14 +47,42 @@ def require_dependencies
           require_relative 'messaging_helper'
         end
 
-        def add_plugin(*targets)
+        def add_plugins(*targets)
           targets.each { |klass| klass.add_plugin(AwsSdk::Plugin) }
         end
 
         def loaded_constants
```

Comment:
Nit: I know this is an existing name, but I find it a little confusing.  I think this is internal only, so maybe we can rename as something like `loaded_service_clients`?